### ISSUE TITLE
constructor ignores network selector, can't actually choose testnet

### ISSUE THREAD

**Issue by lpreimesberger** [wavesplatform/waves-signature-adapter#20]
Docs say:

`SeedAdapter accepts either a seed phrase or an object containing information about the seed phrase on create Object structure: encryptedSeed {string} - seed phrase encoded with passwords password {string} - password that is encrypted seed phrase encryptionRounds {number} - encryption complexity networkCode {string} - network byte code ("W" for mainnet, and "T" for testnet)`

Not so much - if you look at the code it doesn't do anything with networkCode (SeedAdapter.ts:12-22) - this should prolly be split into two interfaces and the generate from text require a network selector.  For now, I need to hard code the network since I don't want to mess with it.  Whee.  :)

**Comment by tsigel** [comment]
Hello! Thanks for your PR!
For change network byte u can use @waves/signature-generator config:
```typescript
import { config } from '@waves/signature-generator';

config.set('networkByte', 'T'.charCodeAt(0))
```

**Comment by tsigel** [comment]
[Fix](https://github.com/wavesplatform/waves-signature-adapter/pull/21/files).

**Comment by enriqueflavio** [comment]
dont work for me...

**Comment by enriqueflavio** [comment]
![image](https://user-images.githubusercontent.com/12115948/57959512-96388180-78da-11e9-8b1a-2b79dc9dd0ea.png)
generating this configuration, is there a problem right?

**Comment by tsigel** [comment]
@enriqueflavio, how u get this log?

**Comment by enriqueflavio** [comment]
96/5000
I'm using 
`config.set ('networkByte', 'T'.charCodeAt (0));`
 and config, stay that way there.

**Comment by tsigel** [comment]
@enriqueflavio, thank u. I'm write wrong code. U need use object for config.

```typescript
import { config } from '@waves/signature-generator';

config.set({ 'networkByte': 'T'.charCodeAt(0) });
```

**Comment by enriqueflavio** [comment]
> @enriqueflavio, thank u. I'm write wrong code. U need use object for config.
> 
> ```ts
> import { config } from '@waves/signature-generator';
> 
> config.set({ 'networkByte': 'T'.charCodeAt(0) });
> ```

Yes, and wrong. The image I submitted shows the config object after this executed method. Looking at the lib code itself, you see that it is a single object that has to be placed as a parameter.
